lib/ruby-next/language/rewriters/safe_navigation.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module RubyNext
+  module Language
+    module Rewriters
+      class SafeNavigation < Base
+        NAME = "safe-navigation"
+        SYNTAX_PROBE = "nil&.x&.nil?"
+        MIN_SUPPORTED_VERSION = Gem::Version.new("2.3.0")
+
+        def on_csend(node)
```

Comment:
I'd suggest doing `node = super` at the beginning of every `on_<something>` method (in all rewriter classes) to make sure that all child nodes are rewritten. Here `args` seems to be ignored and code like `a&.b(a&.c)` is partially transpiled.

Reply:
Makes sense. Thanks!